lms/static/js/student_account/views/AccessView.js

```diff
@@ -6,13 +6,18 @@ var edx = edx || {};
     edx.student = edx.student || {};
     edx.student.account = edx.student.account || {};
 
+    // Bind to StateChange Event
+    History.Adapter.bind(window,'statechange',function(){ // Note: We are using statechange instead of popstate
+        var State = History.getState(); // Note: We are using History.getState() instead of event.state
```

Comment:
Minor, but would it be possible to change these comments so they explain why we're using `statechange` and `History.getState()` instead of `popstate` and `event.state`?